src/system/documents/chat-message.ts

```diff
@@ -17,6 +17,7 @@ export const MESSAGE_TYPES = {
     SKILL: 'skill',
     ACTION: 'action',
     INJURY: 'injury',
+    TAKEN: 'taken',
```

Comment:
Can we rename this to something more clear? I appreciate the shorthand but `taken` feels a bit too abstract.